lib/include/nitro/mi.h

```diff
@@ -1,12 +1,13 @@
-#ifndef NITRO_MI_H_
-#define NITRO_MI_H_
+#ifndef POKEHEARTGOLD_MI_H
```

Comment:
revert

Reply:
why?

Reply:
because these are not gamefreak headers

Reply:
I am aware of that, but as far as I know GF didn't use the POKEHEARTGOLD_<file>_H scheme either, we use it for every other file in the repo, I'm unsure why not here

Reply:
if strictly necessary I can go through and change it to NITRO_MI_H (or NITROSDK_MI_H) but I'd need to change the headers in diamond too